src/components/time-entries/TimeEntries.tsx

```diff
@@ -20,6 +20,8 @@ export const TimeEntries = ({ isModalActive, handleModal }: TimeEntriesProps) =>
   const [ascending, setAscending] = useState(false);
   const [currentClient, setCurrentClient] = useState("");
 
+  console.log(orderedEntries);
+
```

Comment:
You can remove this console.log. 

Tip: as your projects get bigger, it is handy to use the search function and looking for any remaining console logs.

Reply:
Already did in last commit :)